README.md

```diff
@@ -23,7 +22,31 @@ npm install --save-dev @zendeskgarden/tailwindcss
 
 ## Usage
 
-TODO...
+Add the plugin to your `tailwind.config.js`:
+
+```js
+// tailwind.config.js
+module.exports = {
+  plugins: [require('@zendeskgarden/tailwindcss')]
+};
+```
+
+### Configuration
+
+The plugin allows an optional `includeBedrock` configuration which
+will disable Gardens [css-bedrock](https://github.com/zendeskgarden/css-components/tree/master/packages/bedrock)
+global resets.
```

Comment:
re-phrase (with tweak to link): "The plugin provides an optional `includeBedrock` setting which can be used to disable Garden's [css-bedrock](https://github.com/zendeskgarden/css-components/tree/master/packages/bedrock#readme) reset."

Reply:
Updated.